messaging/src/main/java/org/axonframework/queryhandling/SubscriptionQueryBackpressure.java

```diff
@@ -25,6 +25,7 @@
  * @author Milan Savic
  * @since 3.3
  */
+@Deprecated
```

Comment:
The JavaDoc should describe why something is deprecated and what it has been replaced for, if by any. Please add that to the class level here.